### PR TITLE
Adding a v6 version of the mutator sample

### PR DESCRIPTION
**Pull request by andreasohlund**
Note that this sample is still referencing an unstable core version from MyGet. 

To make this work for our users we need to include a `nuget.config` in the downloaded zip. I will create a seprate pull against the docs engine to handle this

### CODE REVIEW

File: samples/encryption/message-body-encryption/Version_6/Endpoint1/app.config [hunk 1]
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (SimonCropp) [inline samples/encryption/message-body-encryption/Version_6/Endpoint1/app.config]: This file is redundant

File: samples/encryption/message-body-encryption/Version_6/Endpoint2/app.config [hunk 2]
```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<configuration>
```
Comment (SimonCropp) [inline samples/encryption/message-body-encryption/Version_6/Endpoint2/app.config]: Redundant file

**Comment by SimonCropp** [comment]
Apart from the redundant files this is good for a merge

**Comment by andreasohlund** [comment]
Ok, I'll remove them, funny I just copy and pasted the v5 sample, wonder
where that app.config came from. Nuget update?

**Comment by SimonCropp** [comment]
@andreasohlund vs adds them when u update to 451. But it is a deployment setting. So useless for samples

**Pull request merged**